common/reactive/src/main/java/io/helidon/common/reactive/OriginThreadPublisher.java

```diff
@@ -219,7 +219,7 @@ public void submit(U data) {
      */
     public void drain() {
         if (!hasSingleSubscriber.get() && !(completed && queue.isEmpty())) {
-            System.out.println("LEAK: no one registered to consume request");
+            LOGGER.fine(() -> "No one registered to consumer request");
```

Comment:
Sorry about that, that change was merged with a PR with I had already merged it in the branch and forgot to clean it up like I did in the corresponding PR.

Reply:
No problem, happens.